packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -104,3 +107,10 @@ describe("interactive-graph widget", function () {
         },
     );
 });
+
+describe("segment graph", () => {
+    it("should render", () => {
+        // Mafs isn't very RTL friendly...
+        renderQuestion(segmentQuestion, blankOptions);
```

Comment:
The `renderQuestion` helper returns the `renderer` itself. So you should be able to call `guessAndScore()` on that to check if the new Mafs grid implementation supports grading correctly. 

We can work on this tomorrow while we pair.